{{cookiecutter.project_slug}}/tests/test_{{cookiecutter.project_slug}}.py

```diff
@@ -26,30 +26,26 @@
 
 
 {% if cookiecutter.use_pytest == 'y' -%}
-class Test{{ cookiecutter.project_slug|title }}(object):
+@pytest.fixture
+def some_fixture():
+    pass
```

Comment:
The documentation lists this example for setup/teardown:

```
import smtplib
import pytest

@pytest.fixture(scope="module")
def smtp(request):
    smtp = smtplib.SMTP("smtp.gmail.com")
    yield smtp  # provide the fixture value
    print("teardown smtp")
    smtp.close()
```

I'm not really sure if we should provide an example of this to the user, or if we should let them read the documentation to figure it out. On IRC, the biggest problem I see with users new to pytest is that they are unable to wrap their head around being able to define functions as fixtures that are then passed as arguments to the test functions. From what I've seen, once they see how fixtures are used, they typically figure out how to do the rest.